LensmaniaClient/Layout/MainLayout.razor.css

```
@@ -0,0 +1,77 @@
+}
+
+.top-row {
+    background-color: #f7f7f7;
+    border-bottom: 1px solid #d6d5d5;
+    justify-content: flex-end;
+    height: 3.5rem;
+    display: flex;
+    align-items: center;
+}
+
+    .top-row ::deep a, .top-row ::deep .btn-link {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: deep

(lint/correctness/noUnknownPseudoElement)